deploy/config

```diff
@@ -7,12 +7,5 @@
 			"opsManagerUserName": "foo",
 			"opsManagerPassword": "bar"
 		}
-	},
-	"targetConfig": {
```

Comment:
Even user can use the default API server address as identifier, should we also provide user a way to specify the target Name(or Identifier)?